### ISSUE TITLE
LCOM5

### ISSUE THREAD

**Issue by yegor256** [yegor256/jpeek#18]
Let's implement LCOM5 cohesion metric.

B. Henderson-Sellers, L. L. Constantine, and I. M. Graham, “Coupling
and cohesion (towards a valid metrics suite for objectoriented
analysis and design),” Object Oriented Systems, vol. 3,
no. 3, pp. 143–158, 1996

http://www.ibrarian.net/navon/paper/Coupling_and_cohesion__towards_a_valid_metrics_su.pdf?paperid=1090060

**Comment by sergey-karazhenets** [comment]
I'm working on it.
More information about LCOM5 calculation is [here](http://www.math.md/files/csjm/v25-n1/v25-n1-(pp44-74).pdf) and [here](https://www.google.by/url?sa=t&rct=j&q=&esrc=s&source=web&cd=3&ved=0ahUKEwjh9PK4nZ7XAhVlEpoKHRMTAS0QFgg2MAI&url=http%3A%2F%2Fwww.ece.rutgers.edu%2F~marsic%2Fbooks%2FSE%2Finstructor%2Fslides%2Flec-16%2520Metrics-Cohesion.ppt&usg=AOvVaw3967dvApLWjTkj6-MBEnkS)

**Comment by yegor256** [comment]
@sergey-karazhenets thanks!

**Comment by yegor256** [comment]
@sergey-karazhenets don't forget to update the README please

**Comment by 0crat** [comment]
Job [#18](https://github.com/yegor256/jpeek/issues/18) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://datum.zerocracy.com/pages/policy.html#29): +15 points just awarded to [@yegor256](http://www.0crat.com/u/yegor256), total is [+7225](http://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#18](https://github.com/yegor256/jpeek/issues/18) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by llorllale** [comment]
@yegor256 FYI, I think there's a typo in their proposed formula on page 157:
![image](https://user-images.githubusercontent.com/2019896/35338671-e90ef4d4-00f4-11e8-96a0-8c1f52a69c3f.png)
I think instead of "m(A<sub>j</sub>)" it should be "μ(A<sub>j</sub>)". Do you agree?

**Comment by yegor256** [comment]
@llorllale I guess, yes

**Comment by llorllale** [comment]
blocked by #103

**Comment by llorllale** [comment]
@0crat waiting

**Comment by 0crat** [comment]
> @0crat waiting ([here](https://github.com/yegor256/jpeek/issues/18#issuecomment-360214290))

@llorllale The impediment for [#18](https://github.com/yegor256/jpeek/issues/18) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by llorllale** [comment]
@yegor256 I believe we can close this

**Comment by yegor256** [comment]
@llorllale thanks!

**Comment by 0crat** [comment]
Order was successfully finished: +30 points just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale), total is [+360](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#18](https://github.com/yegor256/jpeek/issues/18) is now out of scope
